### PR TITLE
Adding 100x reboot tests.

### PR DESCRIPTION
**Pull request by llogen**
Signed-off-by: llogen <christoph.lange@9elements.com>

### CODE REVIEW

File: descriptors/coreboot-qemu-centosX100.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              - ID: "x86_64"
+          MinNumberDevices: 1
+          MaxNumberDevices: 1
+        TargetManagerReleaseParameters:
+        TestFetcherName: literal
+        TestFetcherFetchParameters:
+            TestName: Boot CentOS coreboot
+            Steps:
+              - name: qemu
+                label: qbt
```
Comment (glabelderp) [inline descriptors/coreboot-qemu-centosX100.yaml:18]: We should change this to something like "Qemu Boot Test"

File: descriptors/coreboot-up2-linuxbootX100.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+JobName: Dutctl test
+Runs: 1
```
Comment (glabelderp) [inline descriptors/coreboot-up2-linuxbootX100.yaml:3]: 1 -> 100

**Review by glabelderp — CHANGES_REQUESTED** [review]
Looks good, except for the comments

File: descriptors/coreboot-qemu-centosX100.yaml (hunk 1 above)
Comment (llogen) [reply descriptors/coreboot-qemu-centosX100.yaml:18]: We already did in the other PR. Lets set the wording first and than add new tests.

File: descriptors/coreboot-up2-linuxbootX100.yaml (hunk 2 above)
Comment (glabelderp) [reply descriptors/coreboot-up2-linuxbootX100.yaml:2]: Ah yes forgot about that!

**Review by glabelderp — APPROVED** [review]
LGTM

**Review by glabelderp — APPROVED** [review]

**Pull request merged**